setup.py

```diff
@@ -37,7 +37,9 @@
       install_requires=["wcwidth"],
       setup_requires=["pytest-runner"],
       tests_require=["pytest"],
+      python_requires=">=3.7",
       platforms=["any"],
+      package_data={"pyte": ["py.typed"]},
```

Comment:
Do you need to explicitly add `py.typed` file under `pyte/` or this sufficient?

Reply:
This is required according to PEP 561: https://mypy.readthedocs.io/en/stable/installed_packages.html#creating-pep-561-compatible-packages